src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/ActionButtons.tsx

```diff
@@ -62,25 +81,39 @@ const ActionButtons = ({
 
   return (
     <div className='discovery-modal_buttons-row' data-testid='actionButtons'>
-      <DownloadAllModal
+      <DownloadModal
         downloadStatus={downloadStatus}
         setDownloadStatus={setDownloadStatus}
       />
       <Row className='row'>
-        {/*
-        <Col flex='1 0 auto'>
-          <Button className='discovery-action-bar-button'>
-            Download <br />
-            Variable-Level Metadata
-          </Button>
-        </Col>
-        */}
-        {showDownloadStudyLevelMetadataButtons && (
+        {showDownloadVariableMetadataButton && (
+          <Col flex='1 0 auto'>
+            <Button
+              className='discovery-action-bar-button'
+              disabled={Boolean(
+                downloadStatus.inProgress
+                  || dataDictionaryInfo.noVariableLevelMetadata,
+              )}
+              loading={downloadStatus.inProgress === 'DownloadVariableMetadata'}
+              onClick={() => {
+                DownloadVariableMetadata(
+                  dataDictionaryInfo.dataDictionaries,
+                  resourceInfo,
+                  setDownloadStatus,
+                );
+              }}
+            >
+              Download <br />
+              Variable-Level Metadata
+            </Button>
+          </Col>
+        )}
+        {showDownloadStudyLevelMetadataButton && (
```

Comment:
when user has not logged in, this button should show as `Login to To Download Study-Level Metadata`, and redirect to user to the login page upon clicking, just like the download manifest button